src/Core/src/Core/IEditor.cs

```diff
@@ -11,5 +11,7 @@ public interface IEditor : IView, ITextInput, ITextStyle
 		/// Gets or sets the placeholder text color.
 		/// </summary>
 		Color PlaceholderColor { get; set; }
+
+		TextAlignment HorizontalTextAlignment { get; set; }
```

Comment:
Just implement `ITextAlignment` interface instead include directly the property.